### ISSUE TITLE
Remove "Legacy key" part from Handbook

### ISSUE THREAD

**Issue by os11k** [cardano-foundation/developer-portal#375]
I propose to remove "Legacy key" part from Operate a Stake Pool - Handbook. Do we still need it there?

I found that on 3 pages. 2 of them are in Handbook, so I'm pretty sure nobody needs legacy keys at least for creating a pool. No? If this is ok, I will create PR.

**Comment by katomm** [comment]
This issue is open for a while now, does anyone else have any opinions on this?

**Comment by rdlrt** [comment]
I agree with OP (a very unlikely scenario, but if anyone who does need it has other ways to do so)

**Comment by katomm** [comment]
Since Priyank has confirmed, I am confident it can be removed. Would be great if you can do that @os11k

**Comment by os11k** [comment]
Hi! I'm bit busy this week, I will try to work on this next week on during weekend. Cheers!

**Comment by katomm** [comment]
No rush, sir.
